### PR TITLE
Add New File

### PR DESCRIPTION
**Pull request by kmchmk**
Added a new file. Please review and approve.

### CODE REVIEW

**Comment by andrewson97** [comment]
Looks good to me.

File: chanaka.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is my name.
```
Comment (ghost) [inline chanaka.txt:1]: Add an empty line at the EOF.

Comment (kmchmk) [reply chanaka.txt:1]: Updated.

**Comment by ghost** [comment]
LGTM.

**Review by ghost — APPROVED** [review]

File: chanaka.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+
```
Comment (sarankan-24) [inline chanaka.txt:2]: LGTM

**Review by LahiruKumaraHewagama — APPROVED** [review]
LGTM

**Review by sarankan-24 — APPROVED** [review]
Approved

**Pull request merged**